controllers/profiles/common/app_properties.go

```diff
@@ -38,34 +43,313 @@ import (
 )
 
 const (
-	ConfigMapWorkflowPropsVolumeName = "workflow-properties"
-	kogitoServiceUrlProperty         = "kogito.service.url"
-	kogitoServiceUrlProtocol         = "http"
-	dataIndexServiceUrlProperty      = "mp.messaging.outgoing.kogito-processinstances-events.url"
-	kafkaSmallRyeHealthProperty      = "quarkus.smallrye-health.check.\"io.quarkus.kafka.client.health.KafkaHealthCheck\".enabled"
-	dataIndexServiceUrlProtocol      = "http"
-
-	DataIndexImageBase = "quay.io/kiegroup/kogito-data-index-"
-	DataIndexName      = "data-index-service"
-
-	PersistenceTypeEphemeral   = "ephemeral"
-	PersistenceTypePostgressql = "postgresql"
-
+	ConfigMapWorkflowPropsVolumeName        = "workflow-properties"
+	kogitoServiceUrlProperty                = "kogito.service.url"
+	kogitoServiceUrlProtocol                = "http"
+	dataIndexServiceUrlProperty             = "mp.messaging.outgoing.kogito-processinstances-events.url"
+	kafkaSmallRyeHealthProperty             = "quarkus.smallrye-health.check.\"io.quarkus.kafka.client.health.KafkaHealthCheck\".enabled"
+	dataIndexServiceUrlProtocol             = "http"
+	jobServiceURLProperty                   = "mp.messaging.outgoing.kogito-job-service-job-request-events.url"
+	jobServiceKafkaSinkInjectionHealthCheck = `quarkus.smallrye-health.check."org.kie.kogito.jobs.service.messaging.http.health.knative.KSinkInjectionHealthCheck".enabled`
+	jobServiceStatusChangeEventsProperty    = "kogito.jobs-service.http.job-status-change-events"
+	jobServiceStatusChangeEventsURL         = "mp.messaging.outgoing.kogito-job-service-job-status-events-http.url"
+	jobServiceURLProtocol                   = "http"
+	jobServiceDataSourceReactiveURLProperty = "quarkus.datasource.reactive.url"
+
+	imageNamePrefix                          = "quay.io/kiegroup/kogito"
+	DataIndexServiceName                     = "data-index-service"
+	DataIndexName                            = "data-index"
+	JobServiceName                           = "jobs-service"
+	PersistenceTypeEphemeral                 = "ephemeral"
+	PersistenceTypePostgressql               = "postgresql"
 	microprofileServiceCatalogPropertyPrefix = "org.kie.kogito.addons.discovery."
 	discoveryLikePropertyPattern             = "^\\${(kubernetes|knative|openshift):(.*)}$"
+
+	defaultDatabaseName   string = "sonataflow"
+	defaultPostgreSQLPort int    = 5432
+)
```

Comment:
There are too many properties here that belong to services. Can we please have an app properties handler for workflows and other for the services?

`platform` -> handler for Services
`profiles` -> handler for workflows

We might have a common module that defines a strategy for handling properties, but each module should have its own responsibility accordingly.

Reply:
Done. I'm not sure if I got it right though. There are a lot of inter dependencies in the handler and the platform services that caused a import cycle problem when I tried to keep things together. Eventually I found a way to make a clear cut by defining a package inside `platform` and `profile` that allowed me to call the platform service properties inside the profile handler like it's being done now.

If what you expect is to duplicate the handler struct and functions for `platform` and `profiles`, with the caveat that only in `platform` it will inject the platform services properties, let me know.